dhall-docs/src/Dhall/Docs/Html.hs

```diff
@@ -35,24 +33,10 @@ data DocParams = DocParams
     , packageName :: !String               -- ^ Name of the package
     }
 
--- | Takes a `Header` and generates its `Html`
-headerToHtml :: Header -> Html ()
-headerToHtml = p_ . toHtml . removeComments
-
-removeComments :: Header -> Text
-removeComments (Header header)
-    | "--" `Data.Text.isPrefixOf` strippedHeader = Data.Text.drop 2 strippedHeader
-    | "{-" `Data.Text.isPrefixOf` strippedHeader =
-        Data.Text.drop 2 $ Data.Text.dropEnd 2 strippedHeader
-    | otherwise = strippedHeader
-
-  where
-    strippedHeader = Data.Text.strip header
-
 -- | Generates an @`Html` ()@ with all the information about a dhall file
 filePathHeaderToHtml
-    :: (Path Abs File, Header) -- ^ (source file name, parsed header)
-    -> DocParams               -- ^ Parameters for the documentation
+    :: (Path Abs File, Html ()) -- ^ (source file name, header document as html)
```

Comment:
Nit: Why is this a tuple instead of two separate parameters?

Reply:
I think that at the begginning I passed the whole tuple but it's not like that anymore - I'll split them